src/components/ItemsList/ItemsList.js

```diff
@@ -16,6 +18,9 @@ function ItemsList({ items }) {
             <Item key={item.id} item={item} />
           ))}
         </List>
+        <Content>
```

Comment:
I am not sure how `Content` should be used. Shouldn't it wrap `List` also? @blurbyte can you confirm is it correct?

Reply:
Or maybe that `Content` should be inside `ShippingDetails`?